README.md

```diff
@@ -176,6 +176,34 @@ $ helm dt unwrap kibana-10.4.8.wrap.tgz demo.goharbor.io/helm-plugin/ --yes
 
 If your wrap includes bundled artifacts (if you wrapped it using the `--fetch-artifacts` flag), they will be also pushed to the remote registry.
 
+Unwrapping a Helm chart to a different target than the original using `--push-repository` flag.
```

Comment:
Maybe something like `Unwrapping a Helm chart to a different registry than the one used on relocation` or `Using different registries for distribution and relocation`?

Reply:
It would also be nice to add some context on why this is needed. Something like: 

`Sometimes you might need the content on the Helm chart pointing to a registry different than where the Helm chart needs to be pushed to. This is common for example on Airgap environments where a Helm chart might need to be pushed to an interim environment different from where it is meant to be consumed from, or for example in edge distribution channels where the Helm charts might need to be moved from an interim release registry into an edge registry for general public usage.`